src/sandstorm/seccomp-bpf/clean-header.ekam-rule

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+this_dir="sandstorm/seccomp-bpf"
+
+if test $# = 0; then
+  echo "trigger canonical:${this_dir}/gen-clean-h"
```

Comment:
For a rule that is intended to be executed only once, not on a per-input basis, you can just do the logic when the rule first executes rather than registering a trigger.